src/main/java/com/comcast/drivethru/api/HTTPRequestManager.java

```diff
@@ -418,6 +441,17 @@ public Builder data(byte[] data)
             mData = data;
             return this;
         }
+        
+        /**
+         * Sets request's multipart data.
+         * @param multipart Multipart data to send the server
+         * @return {@link Builder} object
+         */
+        public Builder multipart(HttpEntity multipart)
```

Comment:
That was a worry of mine. Awesome point. I didn't think everyone would know how to use that builder to make an HttpEntity. That's a good place to throw them a bone